src/main/java/workers/CourseCreator.java

```diff
@@ -8,10 +8,11 @@
 import java.time.DayOfWeek;
 import java.time.LocalTime;
 import java.util.ArrayList;
+import java.util.List;
 import java.util.Map;
 
 /**
- * This class represents a entities.Course Creator. This class uses workers.APIWorker to generate
+ * This class represents an entities.Course Creator. This class uses workers.APIWorker to generate
```

Comment:
just write Course rather than entities.Course. Same for the other ones. U can assume since the class names are unique u don't need to specify the packages imo